app/assets/stylesheets/custom/intl_tel_input.css

```diff
@@ -0,0 +1,3 @@
+.iti__flag {
+    background-image: url('https://cdnjs.cloudflare.com/ajax/libs/intl-tel-input/16.0.2/img/flags.png');
```

Comment:
Bunu asset'lerden sunsak daha iyi olmaz mi @isubas? Hem her request icin bu dosyayi cekecek, hem de bu URL'in degisme ihtimali var, zaten URL'de de surum numarasi var.